libs/platform/experience/src/services/layout/plugins/styles/typography/typography-layout.plugin.ts

```diff
@@ -0,0 +1,27 @@
+import { Observable, of } from 'rxjs';
+import {
+  LayoutPlugin,
+  LayoutPluginConfig,
+  LayoutStyleParameters,
+  LayoutStyleProperties,
+} from '../../layout.plugin';
+
+export class TypographyLayoutPlugin implements LayoutPlugin {
+  getConfig(): Observable<LayoutPluginConfig> {
+    return of({
+      schema: () => import('./typography-layout.schema').then((m) => m.schema),
+    });
+  }
+
+  getStyleProperties(
+    data: LayoutStyleParameters
+  ): Observable<LayoutStyleProperties> {
+    if (!data.typography) return of({});
+
+    return of({
+      'font-size': `var(--oryx-typography-${data.typography}-size)`,
+      'font-weight': `var(--oryx-typography-${data.typography}-weight)`,
+      'line-height': `var(--oryx-typography-${data.typography}-line)`,
+    });
+  }
```

Comment:
The typography related model would ideally move out of the `composition.model.ts` and find home here in the model/schema.